content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,546 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+In standalone mode, the Kubelet allows you to manage containers using PodTemplates,
+which brings the benefits of running tightly coupled applications as a single unit while
```

Comment:
I don't think this is accurate. The Kubelet manages pods directly, I don't think there's any support for static PodTemplates.

The main defining feature of standalone mode is that the Kubelet isn't registered with an API server. The usecase you've described is a valid one, but the main use case I've seen for this is running the control plane nodes.

Reply:
Thank you for your review @tallclair. I will rewrite this paragraph accordingly. Could you (as far as you know) elaborate on how the standalone Kubelet mode can be used for running the control plane nodes? We were trying to illustrate more use cases.

Reply:
If you want to run your API server and etcd as pods, there are a bunch of risks to running them as pods in the cluster (the self-hosting approach). A safer way to achieve that is by using a standalone Kubelet to run at least those 2 components

Reply:
Interesting. Thanks

Reply:
Try:
```markdown
In standalone mode, the kubelet allows you to manage containers using Pod _manifests_
```

A detail: those manifests can either be on the local filesystem, or fetched via HTTP from a configuration source.

Reply:
Done